homeassistant/components/sensor/syncthru.py

```diff
@@ -101,7 +107,7 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
         if 'output_tray_{}'.format(key) in monitored:
             devices.append(SyncThruOutputTraySensor(printer, name, key))
 
-    add_entities(devices, True)
+    await async_add_entities(devices, True)
```

Comment:
Does this work? I think `async_add_entities()` is a callback that should not be awaited.

Reply:
right, fixed